jest_after_env_setup.js

```diff
@@ -0,0 +1,15 @@
+// React
+import React from "react";
+global.React = React;
+
+// Jest dom (testing env)
+import "@testing-library/jest-dom";
+
+// Routes
+window.Routes = global.Routes = require("./app/javascript/routes");
+
+// Enzyme configuration
+import {configure} from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
```

Comment:
Is this still needed with Enzyme 3.x? (I just did a quick search so I could be wrong here.)